### DISCUSSION TITLE
copying a language

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1503] [Q&A] [answered]
I created a multi language site and now the customer asks 'can you change the US flag to a UK flag?'
Now, I could just copy the UK flag and rename it to US flag. But I know that in 2 months there is also a US site wanted. 
So, 2 questions:
1. how would you change the flag? Can I replace en-us with en-uk in a way? mind you that I set en-us as site default and system default so I do not want to change 100%, I guess.
2. If I am correct, I could also create en-uk now which would automically create a copy of the default (en-us) right?

**Comment by meetmandeep** [comment, marked as answer]
There is no official way to change the default language in Vanjaro/DNN. Yes. You can enable a new language (en-uk) and that would automatically serve en-us until you create locale specific content.

**Reply by Tychodewaard** [reply]
In the meantime I tried and can confirm.
